### PR TITLE
refactor(feedbackApi): FRMW-379-convert to Quark.

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-379

- [x] Dev LGTM
- [x] Dev LGTM

---
Refactor `configs.feedbackApi` to Quark.

### CODE REVIEW

File: src/quarks/rxFeedbackSvc/scripts/rxFeedbackSvc.spec.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ describe('rxFeedbackSvc', function () {
         };
 
         module(function ($provide) {
-            $provide.value('feedbackApi', apiUrl);
+            $provide.constant('feedbackApi', apiUrl);
```
Comment (100stacks) [inline src/quarks/rxFeedbackSvc/scripts/rxFeedbackSvc.spec.js]: :bangbang: `constant` vs `value`

[[http://giphy.com/gifs/gotham-season-2-watchgotham-online-awVDnZtRK6fJe | alt=constant-vs-value]]

**Comment by CITguy** [comment]
LGTM

File: src/quarks/rxFeedbackSvc/scripts/rxFeedbackSvc.spec.js (hunk 1 above)
Comment (100stacks) [inline src/quarks/rxFeedbackSvc/scripts/rxFeedbackSvc.spec.js]: **Note to Self:** `$provide.constant` vs `$provide.value` :no_mouth: :exclamation:

Comment (halkon) [inline src/quarks/rxFeedbackSvc/scripts/rxFeedbackSvc.spec.js]: fry-not-sure-if

Comment (100stacks) [inline src/quarks/rxFeedbackSvc/scripts/rxFeedbackSvc.spec.js]: @halkon, you'll thank me later for that note :point_up_2: :smile:

Comment (CITguy) [inline src/quarks/rxFeedbackSvc/scripts/rxFeedbackSvc.spec.js]: reverse-stare

**Pull request merged**